spec/features/form_sections/teacher_training_data/apply_applications_spec.rb

```diff
@@ -85,12 +85,13 @@
     let(:subjects) { ["Modern languages (other)"] }
 
     scenario "selecting languages" do
+      ActiveJob::Base.queue_adapter.perform_enqueued_jobs = true
```

Comment:
if this is not the default, this should be set back to false. perhaps a `before` and `after` block outside of the scenario wrapped in a `context` block

Reply:
Removed. Tests work without this.